### PR TITLE
Public Preview of Mixed Reality Cloud Service

### PR DESCRIPTION
**Pull request by XiangyuL-Microsoft**
This is the first check-in of Azure PowerShell module for Mixed Reality Azure Resource Management.
All code had been reviewed and checked-in [private repo](https://github.com/Azure/azure-powershell-pr/tree/Az.MixedReality/src/MixedReality)

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

File: tools/StaticAnalysis/Exceptions/Az.MixedReality/SignatureIssues.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"c:\workspace\ps-pr\artifacts\Debug\Az.MixedReality\Microsoft.Azure.PowerShell.Cmdlets.MixedReality.dll","Microsoft.Azure.Commands.MixedReality.SpatialAnchorsAccount.NewSpatialAnchorsAccountKey","New-AzSpatialAnchorsAccountKey","1","8510","Cmdlet 'New-AzSpatialAnchorsAccountKey' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```
Comment (cormacpayne) [inline tools/StaticAnalysis/Exceptions/Az.MixedReality/SignatureIssues.csv]: @XiangyuL-Microsoft for this issue of `New-AzSpatialAnchorsAccountKey` not having a default parameter set, how do you feel about introducing a set that doesn't require `-Primary` or `-Secondary` and by default, regenerates both keys for the user?

File: src/MixedReality/MixedReality/Az.MixedReality.psd1 [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+DotNetFrameworkVersion = '4.7.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.3.0'; })
```
[the remark this answers is not in the document]
Comment (XiangyuL-Microsoft) [reply src/MixedReality/MixedReality/Az.MixedReality.psd1]: Offline synced, it is 1.3.0 now

File: src/MixedReality/MixedReality/SpatialAnchorsAccount/GetSpatialAnchorsAccountKey.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+            if (ParameterSetName == PipelineParameterSet)
+            {
+                ResourceGroupName = InputObject.ResourceGroupName;
+                Name = InputObject.Name;
+            }
+
+            var result = Client.SpatialAnchorsAccounts.GetKeys(ResourceGroupName, Name);
+
+            WriteObject(result);
```
[the remark this answers is not in the document]
Comment (XiangyuL-Microsoft) [reply src/MixedReality/MixedReality/SpatialAnchorsAccount/GetSpatialAnchorsAccountKey.cs]: Yes, it is designated to be an object against a list.

File: tools/StaticAnalysis/Exceptions/Az.MixedReality/SignatureIssues.csv (hunk 1 above)
Comment (XiangyuL-Microsoft) [reply tools/StaticAnalysis/Exceptions/Az.MixedReality/SignatureIssues.csv]: When working on the private repo, I have a discussion with @vladimir-Shcherbakov. This is by design. Two keys are provided to make sure even one of them is regenerated, the other can still work. Users are not expected to re-generate both of them simultaneously, So no default parameter set is provided for this Cmdlet to force user explicitly specifying which key to be regenerated.

**Pull request merged**